src/jdk.compiler/share/classes/com/sun/tools/javac/parser/JavaTokenizer.java

```diff
@@ -1062,18 +1062,30 @@ public Token readToken() {
                     next();
 
                     if (accept('/')) { // (Spec. 3.7)
-                        skipToEOLN();
+                        if (accept('/')) { // Markdown comment
```

Comment:
Here and elsewhere in this file: do we need to mention Markdown?

Reply:
The "M" word appears 10 times in this file. I'll work to convert them to an alternate nomenclature, such as "line comment".